product-service/createnewproducthandler/createnewproducthandler.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+const { Client } = require("pg");
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbOptions = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+  connectionTimeoutMillis: 5000,
+};
```

Comment:
WDYT? Maybe better to take away dbOptions in common config file and import it in lambda functions? It will help avoid code dublications.